ManageIQ/manageiq-ui-classic
# [GAPRINDASHVILI] Display custom buttons after comming from relationship table

The code displays the custom buttons if the previous screen was a
provider screen (`@record`) and displayed item (`@display`) is from the list
of classes supported by custom buttons (`APPLIES_TO_CLASS_BASE_MODELS`)

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1635738

Links 
----------------
* https://bugzilla.redhat.com/show_bug.cgi?id=1635738

Steps for Testing/QA
-------------------------------

* Create a custom button for any entity
* Navigate to the entity through the provider relationship screen
* Custom button should be visible
